client/dashboard/sites/scan/notices.tsx

```diff
@@ -0,0 +1,86 @@
+import { JETPACK_CONTACT_SUPPORT } from '@automattic/urls';
+import { Button, ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useState, useEffect } from 'react';
+import { Notice } from '../../components/notice';
+import type { ScanStatusType } from './use-scan-state';
+
+interface ScanNoticesProps {
+	status: ScanStatusType;
+	threats: number;
```

Comment:
should we use `threatCount` instead? Similar to this comment: https://github.com/Automattic/wp-calypso/pull/105906/files#r2364384590

`threats` could be confused with an array of threat like [we use here](https://github.com/Automattic/wp-calypso/blob/97220d0d483d439a84270dd48d51e42ad0d81beb/client/dashboard/sites/scan/components/threats-detail-card.tsx#L14).